src/Scripting/Export/Archive.cpp

```diff
@@ -39,6 +39,7 @@
 #include "Utility/StringUtils.h"
 #include "thirdparty/sol/sol.hpp"
 
+
```

Comment:
Please remove this useless change.

Reply:
I was testing something and forgot that. I'll fix that and the .DS_STORE files